app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -152,9 +152,7 @@ export default {
         componentOptions: {
           simplefile: {
             config: Vue.prototype.$config,
-            headers: {
-              Authorization: `Bearer ${this.token}`,
-            },
+            tokenFromChefs: this.getCurrentAuthHeader
```

Comment:
Style: Better to name this chefsToken (active vs passive tense)

Reply:
Good idea